fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -373,12 +374,15 @@ private FileCacheValue loadFiles(FileCacheKey key) {
         }
     }
 
-    private JobConf getJobConf() {
-        Configuration configuration = new HdfsConfiguration();
-        for (Map.Entry<String, String> entry : catalog.getCatalogProperty().getHadoopProperties().entrySet()) {
-            configuration.set(entry.getKey(), entry.getValue());
+    private synchronized JobConf getJobConf() {
+        if (jobConf == null) {
+            Configuration configuration = new HdfsConfiguration();
+            for (Map.Entry<String, String> entry : catalog.getCatalogProperty().getHadoopProperties().entrySet()) {
+                configuration.set(entry.getKey(), entry.getValue());
```

Comment:
There is a problem, that if we alter the catalog's properties, it will never refresh.
So we need to refresh this jobConf when refreshing the catalog or altering the catalog